src/bin/snapshots/snapshots__ModuleContents.hello-world.input.snap

```diff
@@ -0,0 +1,25 @@
+---
+created: "2019-04-25T18:00:11.876980400Z"
+creator: insta@0.7.4
+source: src/bin/snapshots.rs
+expression: forest
+---
+Ok(
+    _ => ModuleContents {
+        attrs: _ => [],
+        items: _ => [
+            _ => Item {
+                attrs: _ => [],
+                kind: _ => ItemKind::Fn {
+                    header: _ => FnHeader,
+                    decl: _ => FnDecl {
+                        name: _
+                    },
+                    body: _ => Block {
+                        attrs: _ => []
```

Comment:
Shouldn't there be more stuff here?

Reply:
uh, yes... but [we didn't give the statements a name](https://github.com/rust-lang-nursery/wg-grammar/blob/272a8c651478a81929b6e6010a0634931d17e550/grammar/stmt.lyg#L8) so they're not accessible 🙃

Reply:
Can you opportunistically fix that here? =D

Reply:
Or... you can file a separate PR 😂